15. API/task2_API_requests/pogoda.py

```diff
@@ -0,0 +1,33 @@
+import requests
+from pprint import pprint
+
+while True:
+    city = input("Введіть назву міста: ")
+
+    # Отримання координат міста за допомогою API open-meteo.com
+    response = requests.get(f'https://geocoding-api.open-meteo.com/v1/search?name={city}')
+    if response.status_code == 200:
+        data = response.json()
+        # pprint(data)
+        # print(data['results'][0]['latitude'])
+
+        if 'error' in data:
+            print(f"Помилка: {data['error']['message']}")
+        else:
+            lat, lon = data['results'][0]['latitude'], data['results'][0]['longitude']
+            # Отримання поточних показників погоди за допомогою API open-meteo.com
+            response = requests.get(f'https://api.open-meteo.com/v1/forecast?latitude={lat}&longitude={lon}&current_weather=true')
```

Comment:
nit: better to create separate functions for all steps of getting information. Too much if blocks here